config/localization.yaml

```diff
@@ -1,9 +1,9 @@
 world_frame: "map"
 odom_frame: "odom"
 rover_frame: "base_link"
-use_odom_frame: false
+use_odom_frame: true
```

Comment:
lets not default this to true in this PR, we should wait till we test it with nav